content/topics/free-code-camp/javascript-data-structures-and-algorithms/7-object-oriented-programming/_index.md

```diff
@@ -2,21 +2,21 @@
 _db_id: 312
 content_type: project
 flavours:
-- none
+  - none
 ready: true
 submission_type: link
 tags:
-- javascript
-- oop
-title: FreeCodeCamp - Object Orientated Programming
+  - javascript
+  - oop
+title: FreeCodeCamp - Object Oriented Programming
 ---
 
 Make an account here if you haven't already: https://www.freecodecamp.org/learn
 
-Go to the Menu and select Curriculum, under "JavaScript Algorithms and Data Structures," complete the sections "Build JavaScript Objects" to "Accessing Nested Objects".
+Under "JavaScript Algorithms and Data Structures," complete the all challenges under "Object Oriented Programming".
```

Comment:
I think this "Go to the Menu and select Curriculum, under " is still essential to make navigation easier for some learners who might not know where the stuff is on the website

Reply:
You're automatically navigated to the curriculum page when you first sign in. But I guess it doesn't hurt to be specific. Will change it back.